project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.16-M1
+sbt.version=0.13.16
```

Comment:
I guess we can't use SBT 1.0 to build the plugin itself, since then we wouldn't be able to back-publish.

Reply:
When I originally put this together with SBT 1.0.0 that was the case. Looks like most recent SBT supports cross-building though — I'll submit a quick PR here shortly.